### DISCUSSION TITLE
[Issue] Order is on-hold but products are still in stock

### DISCUSSION THREAD

**Discussion by josephelkhoury** [Blair2004/NexoPOS#1385] [Releases]
**Describe the bug**
When an order is on-hold (customer received the products but didn't pay yet), inventory is not affected.

**Expected behavior**
Stock should be updated accordingly, no?

**Environment:**
 - NexoPOS 4.x version: 4.8.15

**Comment by Blair2004** [comment]
Hi,
Consider a hold order as paused order. It's not an order that has no payment but that the customers plans to pay later. Such order shouldn't hold the inventory. Usually, this used if the customer is undecided or would like to do something else before completing the order. Nothing says he'll come back. Meanwhile, the goods should remains available for other customers.>

NexoPOS put the inventory aside when the order is likely to receive a payment from a customer (either through instalment or with a full-payment) I'm open having discussion that makes me consider a different way of doing.

This will be moved to discussion, as it's intentional and therefore not a bug.

**Reply by josephelkhoury** [reply]
Let's say we're in a restaurant and the customer arrives for lunch and sits at a table.
He orders what he wants and will pay at the end. 
The order is created, items sent to the kitchen for cooking, order is obviously put on hold since no payment was done. 
Inventory in this case shouldn't be updated?
Is there another way around?
